addons/crm/static/src/js/tours/crm.js

```diff
@@ -1,100 +1,113 @@
+    content: _t("<b>Create your first opportunity.</b>"),
+    position: 'bottom',
 }, {
-    trigger: ".o_kanban_quick_create input:first",
-    content: _t("<b>Choose a name</b> for your opportunity, example: <i>'Need a new website'</i>"),
+    trigger: ".o_kanban_quick_create .o_field_widget[name='partner_id']",
+    content: _t('Look for a Contact or create a new one.<br><i>Tip : Did you know you can search by VAT Number as well?</i>'),
     position: "right",
+    run: function (actions) {
+        actions.text("Brandon Freeman", this.$anchor.find("input"));
+    },
+}, {
+    trigger: ".o_kanban_quick_create .o_field_widget[name='name']",
+    auto: true,
+}, {
+    trigger: '.o_kanban_quick_create .o_field_monetary[name="planned_revenue"]',
+    content: _t("Define here the Expected Revenue of this Opportunity."),
+    position: 'right',
 }, {
     trigger: ".o_kanban_quick_create .o_kanban_add",
-    content: _t("Click here to <b>add your opportunity</b>."),
+    content: _t("Now, <b>add your Opportunity</b> to your Pipeline."),
     position: "bottom",
 }, {
-    trigger: ".o_opportunity_kanban .o_kanban_group:first-child .o_kanban_record:last-child",
+    trigger: ".o_opportunity_kanban .o_kanban_group:first-child .o_kanban_record:last-child .oe_kanban_content",
+    extra_trigger: ".o_opportunity_kanban",
     content: _t("<b>Drag &amp; drop opportunities</b> between columns as you progress in your sales cycle."),
-    position: "right",
+    position: "bottom",
     run: "drag_and_drop .o_opportunity_kanban .o_kanban_group:eq(2) ",
 }, {
     trigger: ".o_kanban_record:not(.o_updating) .o_activity_color_default",
     extra_trigger: ".o_opportunity_kanban",
-    content: _t("This opportunity has <b>no activity planned</b>."),
-    position: "bottom"
+    content: _t("Looks like nothing is planned. :(<br><br><i>Tip : Schedule activities to keep track of everything you have to do!</i>"),
+    position: "bottom",
 }, {
     trigger: ".o_schedule_activity",
     extra_trigger: ".o_opportunity_kanban",
-    content: _t("Let's schedule an activity."),
-    position: "bottom"
+    content: _t("Let's <b>Schedule an Activity.</b>"),
+    position: "bottom",
+    width: 200,
 }, {
-    trigger: '.modal-body .o_field_many2one',
-    extra_trigger: ".o_opportunity_kanban",
-    content: _t("Choose an activity type.<br/>You can customize them in the general settings."),
+    trigger: '.modal-footer button[name="action_close_dialog"]',
+    content: _t("All set. Let’s <b>Schedule</b> it."),
     position: "bottom",
     run: function (actions) {
         actions.auto('.modal-footer button[special=cancel]');
     },
 }, {
+    trigger: ".o_opportunity_kanban .o_kanban_record:last-child",
+    content: _t("Drag your opportunity to <b>Won</b> when you get the deal. Congrats !"),
+    position: "bottom",
+    run: "drag_and_drop .o_opportunity_kanban .o_kanban_group:eq(3) ",
+}, ...new CrmAdditionalTourSteps()._get_crm_iap_lead_steps(), {
```

Comment:
Now that I'm seeing this, it's not very modular.
The "crm" app should not contain methods that are clearly designed for another app ("crm_iap_lead").

In "crm_iap_lead", you should insert your extra steps in this tour instead.
You can find an example at: addons/website_sale_product_configurator/static/tests/tours/website_sale_buy.js